### PR TITLE
Hide brand price profiles

### PR DESCRIPTION
**Pull request by R0MANDEV**
#### Type Of Change 
- [x] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Review by ironArt3mis — CHANGES_REQUESTED** [review]
Functional review required

**Review by rbatistadev — APPROVED** [review]

**Comment by rbatistadev** [comment]
I can't see the logic indicating that "if a brand doesn't have tariff..., I cannot show it on the list"

File: web/portal/brand/src/entities/Company/Company.tsx [hunk 1]
```diff
@@ -366,6 +372,23 @@ const properties: CompanyProperties = {
...
 };
 
+export const ChildDecorator: ChildDecoratorType = (props) => {
+  const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+  const companyFeatures = aboutMe?.features;
+
+  const { routeMapItem } = props;
+  if (
+    isEntityItem(routeMapItem) &&
+    routeMapItem.entity.iden === RatingProfile.iden
```
Comment (rbatistadev) [inline web/portal/brand/src/entities/Company/Company.tsx:382]:
```suggestion
    isRatingProfile
```

File: web/portal/brand/src/entities/Company/Company.tsx [hunk 2]
```diff
@@ -366,6 +372,23 @@ const properties: CompanyProperties = {
...
+  ) {
+    if (!companyFeatures?.includes('billing')) {
```
Comment (rbatistadev) [inline web/portal/brand/src/entities/Company/Company.tsx:384]:
```suggestion
    if (!containsBillingFeature) {
```

**Review by Kaian — APPROVED** [review]

**Review by ironArt3mis — APPROVED** [review]

**Pull request merged**